client/client_test.go

```diff
@@ -614,51 +612,6 @@ func TestConcurrentIncrements(t *testing.T) {
 	}
 }
 
-// TestClientPermissions verifies permission enforcement.
```

Comment:
unfortunately not. authentication for RPC happens at connection-negotiation time, so there is nowhere to hook in and do any permission checking per-method.